TipBot/Helpers/FullNodeApiClient.cs

```diff
@@ -0,0 +1,127 @@
+﻿using System;
+using NLog;
+using System.Net.Http;
+using System.Collections.Generic;
+using Newtonsoft.Json;
+using System.Text;
+using System.Web;
+
+namespace TipBot.Helpers
+{
+	public class FullNodeApiClient
```

Comment:
Comments for the whole class: 

1) Use `this.` when accessing fields and class variables. 
2) Don't violate var codestyle. If type is on the right- use var. Otherwise use explicit type. 
3) Add comments
4) Add logs
5) Add spaces between lines. `}` characters should always be followed by an empty line. 
6) All method's names should start with a capital letter.